features/page_objects/sections/expense_section.rb

```diff
@@ -12,7 +12,7 @@ class ExpenseSection < SitePrism::Section
   element :amount, "#claim_expenses_attributes_0_amount"
   element :vat_amount, "#claim_expenses_attributes_0_vat_amount"
 
-  section :expense_date, "fieldset#expense_1_date" do
+  section :expense_date, "div#expense_1_date" do
```

Comment:
**[not a blocker]** is there any point in leaving changing the DOM element every time given this section is already identified by and ID anyway? I would consider removing the DOM element all-together and leave the section as  "#expense_1_date" only, preventing further changes like this one.